sdks/java/io/file-based-io-tests/pom.xml

```diff
@@ -139,6 +139,24 @@
                 </plugins>
             </build>
         </profile>
+        <profile>
+            <!-- Include the google-cloud-platform activated by -Dfilesystem=GCS
+            Support for protocol scheme gs:// - allow to read/write to google storage -->
+            <id>google-cloud-storage</id>
+            <activation>
+                <property>
+                    <name>filesystem</name>
+                    <value>GCS</value>
```

Comment:
Does this require GCS to be all caps ? If so is there a way to not require that ?

Reply:
When provided -Dfilesystem=gcs it won't activate this profile. We should make decision whether uppercased or lowercased value of property is better.

Reply:
Will one of the solutions mentioned here work - https://stackoverflow.com/questions/10521860/property-autocapitalization-in-maven ?